Examples/Notebooks/Circuits/SP_Validation_ReducedOrderSG_VBR_SMIB_Fault.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Validation of the 3th, 4th and 6th Order VBR SynGen against PSAT"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example is taken from: P. Kundur, \"Power System Stability and Control\", Example 13.2, pp. 864-869."
```

Comment:
Is this correct? In [SP_ReducedOrderSG_SMIB_Fault](Examples/Cxx/Circuits/SP_ReducedOrderSG_SMIB_Fault.cpp) you are using according to [this](https://github.com/sogno-platform/dpsim/blob/3dc0f557d3d55ed6dc06f9333ee326fe3e59dc44/Examples/Cxx/Circuits/SP_ReducedOrderSG_SMIB_Fault.cpp#L12) line the `555MW` machine (`H=3.7`), instead of the `2220MW` machine (`H=3.5`), which is defined under the here written Kundur reference according to [Examples.h](https://github.com/sogno-platform/dpsim/blob/3dc0f557d3d55ed6dc06f9333ee326fe3e59dc44/Examples/Cxx/Examples.h#L89).